packages/common/src/store/pages/chat/selectors.ts

```diff
@@ -254,13 +256,24 @@ export const getCanCreateChat = createSelector(
       }
     }
 
+    // Check for existing chat, since unblocked users with existing chats
```

Comment:
No, I intentionally said unblocked because blocked users still can't message

Reply:
sry still confused - if the user is blocked/blocker, wouldn't they get filtered out on line 273/274? And on 275 aren't we checking permissions (not blocker/blockee)?
are you referring to the case where someone was blocked and then unblocked? bc i thought this was referring to the case where user A changed their perms, but if they had a previously existing chat with user B, B would still be able to message A.

Reply:
Yep that's exactly correct! Maybe my wording here is awkward? I'm trying to say that we check for existing chats in the case where the user is not blocked by/is not blocking the other user, since they don't need permission if they already have a chat (and aren't blocked by or blocking the other user)